src/main/java/mServer/crawler/sender/br/BrSendungDetailsTask.java

```diff
@@ -0,0 +1,93 @@
+
+  private final transient MediathekReader crawler;
+  private final ConcurrentLinkedQueue<String> filmIds;
+  private final transient Set<DatenFilm> convertedFilms;
+
+  public BrSendungDetailsTask(final MediathekReader aCrawler,
+      final ConcurrentLinkedQueue<String> aBrFilmIds) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aBrFilmIds" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)